examples/ticks_and_spines/tick-formatters.py

```diff
@@ -51,22 +51,23 @@ def setup(ax):
         fontsize=15, transform=ax.transAxes)
 
 
-# Func formatter
+# FuncFormatter can be used as a decorator
+@ticker.FuncFormatter
 def major_formatter(x, pos):
     return "[%.2f]" % x
 
 
-ax = plt.subplot(n, 1, 3)
+ax = fig.add_subplot(n, 1, 3)
 setup(ax)
 ax.xaxis.set_major_locator(ticker.MultipleLocator(1.00))
 ax.xaxis.set_minor_locator(ticker.MultipleLocator(0.25))
-ax.xaxis.set_major_formatter(ticker.FuncFormatter(major_formatter))
+ax.xaxis.set_major_formatter(major_formatter)
```

Comment:
I missed the PR, but I think if you really want to suggest using decorator syntax, you can do even better with
```
@ax.xaxis.set_major_formatter
@ticker.FuncFormatter
def _(x, pos):
    return ...
```